timetracker/timetracker/urls.py

```diff
@@ -16,11 +16,47 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.http import HttpResponse
+from django.views.decorators.http import require_http_methods
 
+# If there's no name, display a form asking for name.
+# This is the HTML for that form.
+form_page_html = """
+  <html>
+    <body>
+      <p>Hi, what's your name?</p>
 
+      <form method="GET">
```

Comment:
A couple of small tips on your HTML here:

- you don't actually need `method="GET"` here, since that's the default.
- although it's not technically required, a `<input type="submit">` wouldn't go astray here :stuck_out_tongue: